src/modules/events/EventBody.tsx

```diff
@@ -0,0 +1,20 @@
+
+type EventBodyProps = {
+    organizerName: string;
+    date: string;
+    attendeesCount: number;
```

Comment:
I think we can make one common type for `Event` and then reuse it on `EventHeader` and `EventBody`. Currently, you are repeating most of the stuffs from `EventCardProps` into `EventBodyProps` and `EventHeaderProps`

Reply:
Good one! I didn't know we could do that. Code looks much cleaner now! thank you for pointing this out.